packages/kotti-ui/source/kotti-breadcrumb/KtBreadcrumb.vue

```diff
@@ -79,6 +89,9 @@ export default defineComponent<KottiBreadcrumb.PropsInternal>({
 				cursor: pointer;
 			}
 		}
+		&__title--is-active {
+			border-bottom: 1px solid var(--breadcrumb-color-active);
```

Comment:
Using border-bottom only when active moves the item up a bit. You should consider combining this with `margin-bottom: -1px` or with always having a border and changing it from transparent to visible.

Reply:
@HansAuger @FlorianWendelborn I disregard your comments since the design input is to make it colored with var(--text-01) when the step is active